pkg/controllers/user/logging/utils/configmap.go

```diff
@@ -38,22 +38,22 @@ func InitConfigMap(cm rv1.ConfigMapInterface) error {
 func UpdateConfigMap(configPath, loggingName, level string, configmaps rv1.ConfigMapInterface) error {
 	configMap, err := buildConfigMap(configPath, loggingconfig.LoggingNamespace, loggingName, level)
 	if err != nil {
-		return errors.Wrap(err, "buildConfigMap failed")
+		return errors.Wrapf(err, "build configmap %s:%s failed", loggingconfig.LoggingNamespace, loggingName)
 	}
-	existConfig, err := configmaps.Controller().Lister().Get(loggingconfig.LoggingNamespace, loggingName)
+	existConfig, err := configmaps.Get(loggingName, metav1.GetOptions{})
```

Comment:
Why not use Cache here?

Reply:
Checked with Michelia, the object is used for update, which should not use the cache as it may not be the lastest version and update will fail. This is a common pattern in other code.

Reply:
@zionwu even then it can be used from cache. If the update will fail, the sync will be retried.